modules/transactions.js

```diff
@@ -102,6 +102,8 @@ __private.list = function(filter, cb) {
 	const params = {};
 	const where = [];
 	const allowedFieldsMap = {
+		senderIdOrRecipientId:
+			'"t_senderId" IN (${senderIdOrRecipientId:csv}) OR "t_recipientId" IN (${senderIdOrRecipientId:csv})',
```

Comment:
One test case you should add is to check `:csv` formatter. Means send comma separated string value of this param. I believe it will fail because of `address` format we specified in swagger. So either you have to create new format for it or drop `:csv` here.